packages/static-build/src/index.ts

```diff
@@ -48,7 +48,7 @@ import {
 import { getHugoUrl } from './utils/hugo';
 import { once } from 'events';
 
-const SUPPORTED_RUBY_VERSION = '3.2.0';
+const SUPPORTED_RUBY_VERSION = '3.3.0';
```

Comment:
I didn't dig into this, but we probably need this value to change based on the build image (node 20 or not) we're using.

Ref: https://github.com/vercel/vercel/blob/add-ruby33-to-builder/packages/static-build/src/index.ts#L582

Reply:
It's not really critical. This value is used to determine the vendor path where the Gem deps will be installed to. If anything, we should revisit this to install into `.vercel/cache` instead.